doc/releases/changelog-dev.md

```diff
@@ -296,6 +296,8 @@
   and the diagonalising gates using the factors/summands instead of using the full matrix.
   [(#3022)](https://github.com/PennyLaneAI/pennylane/pull/3022)
 
+* `qml.grouping.is_pauli_word` now returns `False` for non-Observables, instead of raising an error.
```

Comment:
```suggestion
* `qml.grouping.is_pauli_word` now returns `False` for non-Observables, instead of raising an error.
  [(#3039)](https://github.com/PennyLaneAI/pennylane/pull/3039)
```